docs/pxe/maas.md

```diff
@@ -0,0 +1,172 @@
+The name of the new machine will be auto-generated from an adjective and an animal, for example "alert-pigeon".
+
+Click the name of the new machine, and this will navigate to a page for this machine.
+It should show several auto-detected details about the machine, such as the nymber of CPUs, the memory, and the storage available on the machine.
+If you want to, you can also rename the machine by clicking the name at the top of the page.
+
+Before we can deploy this machine, we have to configure it so that MAAS can turn it on or off (or tell MAAS that we will do this manually).
+
+1. On the page for the machine, look for the navigation bar *under* the machine's name.
+1. Click Configuration.
+1. Scroll down until you see the Power configuration setting.
+    If this has already been selected to a pre-configured value, you're good to go, because MAAS has autodetected how to control the power on this node!
+1. If it's not yet selected, click the drop-down menu and it will show several different types of power control available.
+    For a physical machine, the most common choice will be "IPMI". 
+    MAAS also supports power control via several different types of hypervisors, such as VMWare.
+    If none of these are available for your hypervisor, click "Manual". This will allow you to proceed with deploying your node, but you will have to manually boot it up when needed.
+1. Click "Save changes"
+
+### Commission the test VM
+
+Once the machine is configured for power control, you will need to *commission* the machine.
+This process is similar to enlistment, but prepares the machine to have an OS deployed.
+
+Navigate to the machine page for your new machine, then click the green "Take action" button in the upper-right hand corner.
+Click "Commission", and then click the green "Commission" button to confirm.
+If you selected manual power control, you should now turn the machine on.
+
+At this point, the machine will once again PXE boot from the MAAS server and run commissioning scripts.
+Once these are finished it will once again turn itself off.
+
+### Deploying the test VM
+
+Once the machine has completed commissioning, navigate to the machine page for your new machine, then click the green "Take action" button.
+Click "Deploy", then confirm by clicking "Deploy machine".
+If you selected manual power control, you should now turn the machine on.
+
+At this point, the machine will PXE boot from the MAAS server and begin installing the OS to the local disk.
+After the installation completes, the machine should show as "Deployed" in the MAAS web interface.
+And at this point your should be able to log in using the SSH key you configured in MAAS!
+
+## Scaling up
+
+This is a lot of work for a single machine, but most operations in MAAS can be done in batches.
+It's generally straightforward to auto-discover a large number of hosts at a time,
+then commission them and provision them by selecting them as a group in the Machines interface.
```

Comment:
Might be useful to add to the "Deploying the test VM" section (or create a new section) which explains that when you wish to reset a system, it isn't necessary to run through commissioning, etc - just deploy once again with the desired OS settings.